### DISCUSSION TITLE
How to share cookies between multiple services

### DISCUSSION THREAD

**Discussion by Wzhipeng** [zauberzeug/nicegui#2586] [Q&A]
### Description

I deploy the application in multiple processes and use `app.storage.user.update({'username': username, 'authenticated': True})` to manage whether the user has successfully logged in. From the nicegui documentation, the data of **app.storage.user** is the cookies that exist on the client side.
But I found that this does not seem to be the case. For example, if I deploy 10 services, if I connect to a new process, I still need to re-authenticate. How to solve it？

**Comment by falkoschindler** [comment]
Hi @Wzhipeng,

This looks like feature request https://github.com/zauberzeug/nicegui/discussions/1606, which is still not implemented.

**Reply by Wzhipeng** [reply]
Hi @falkoschindler  Is this feature in the plan?

**Reply by rodja** [reply]
It's not something planned for the next 1-3 month I fear. Besides relying on the community we recently introduced a way to [sponsor us](https://github.com/sponsors/zauberzeug?frequency=one-time), so we could spent more time on NiceGUI.
